docs/guides/Quick Start/02-product-management.md

```diff
@@ -0,0 +1,547 @@
+# Product Overview
```

Comment:
Any particular reason, why this is called "Product Overview" instead of "Product Management"?

Reply:
As we started with basic info, I add as Overview. But i agree that there is no concrete reason why it has to be written that way when we dont have lower levels to describe.